src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.scss

```diff
@@ -0,0 +1,31 @@
+@use './src/styles/vendor/variables' as vv;
+
+.danger-icon {
+  color: vv.$danger;
+}
+
+.warning-icon {
+  color: vv.$warning;
+}
```

Comment:
By moving this to a top-level SCSS you can reuse this in the other Waad's PR that also displays a warning icon.